lib/node_modules/@stdlib/stats/base/dists/README.md

```diff
@@ -124,7 +128,22 @@ var distributions = dists;
 var objectKeys = require( '@stdlib/utils/keys' );
 var dists = require( '@stdlib/stats/base/dists' );
 
-console.log( objectKeys( dists ) );
+var arcsine = dists.arcsine;
+var beta = dists.beta;
+var normal = dists.normal;
+
+// Create instances of specific distributions
+var myArcsine = arcsine.factory(0.0, 1.0);
+var myBeta = beta.factory(2.0, 2.0);
+var myNormal = normal.factory(0.0, 1.0);
+
+// Generate random numbers
```

Comment:
The packages in this namespace cannot be used to generate random numbers. PRNG APIs are in a different namespace.